docs/source/executing-operations/demand-control.mdx

```diff
@@ -0,0 +1,554 @@
+
+After an operation is executed, the actual cost per operation can be calculated with the actual list sizes.
+
+<Note>
+
+The difference between estimated and actual operation cost calculations is due only to the difference between assumed and actual sizes of list fields.
+
+</Note>
+
+### Measurement and enforcement modes
+
+When rolling out demand control, you first need to gather information about the queries that are already being executed against your graph so you can decide when to reject requests.
+
+The router's demand control supports a measurement mode that enables you to gather this information without impacting your running services. You can define telemetry instruments to monitor your operations and decide on their maximum cost threshold.  
+
+After gathering enough data, you can then configure your router with maximum cost and list size limits and set demand control to enforcement mode, where it rejects operations with costs exceeding the limit.
+
+## Configuring demand control
+
+To enable demand control in the router, configure the `preview_demand_control` option in `router.yaml`:
+
+```yaml title="Example demand control configuration"
+preview_demand_control:
+  enabled: true
+  mode: measure
+  strategy:
+    static_estimated:
+      list_size: 10 
+      max: 1000 
+```
+
+When `preview_demand_control` is enabled, the router measures the cost of each operation and can enforce operation cost limits, based on additional configuration.  
+
+Customize `preview_demand_control` with the following settings:
+
+| Option              | Valid values            | Default value | Description                                                                                          |
+| ------------------- | ----------------------- | ------------- | ---------------------------------------------------------------------------------------------------- |
+| enabled             | boolean                 | `false`       | Set `true` to measure operation costs or enforce operation cost limits.                              |
+| mode                | `measure`, `enforce`    |               | - `measure` collects information about the cost of operations.<br/>- `enforce` rejects operations exceeding configured cost limits |
+| strategy            | `static_estimated`      |               | `static_estimated` estimates the cost of an operation before it is sent to a subgraph                 |
+| static_estimated.list_size  | integer                 |               | The assumed maximum size of a list for fields that return lists.                                      |
+| static_estimated.max        | integer                 |               | The maximum cost of an accepted operation. An operation with a higher cost than this is rejected. |
+
+
+## Telemetry for demand control
```

Comment:
In this section, we should have a link out to more documentation about advanced telemetry. Anybody not familiar with the feature can find this section a little confusing

Reply:
added tip with link